client/my-sites/plans-single-products/index.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from 'config';
+import Product from './product';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const PlansSingleProducts = ( { translate } ) => {
+	const displayBackup = isEnabled( 'plans/jetpack-backup' );
+	const displayScan = isEnabled( 'plans/jetpack-scan' );
+
+	if ( ! displayBackup && ! displayScan ) {
```

Comment:
Is this check necessary? It seems like we're checking in the parent component anyway.

Reply:
Good catch! It was here before I added a similar check to the parent component. I'll get rid of this check in the next PR.